src/main/java/com/aws/greengrass/clientdevices/auth/metrics/MetricsEmitter.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.clientdevices.auth.metrics;
+
+import com.aws.greengrass.telemetry.PeriodicMetricsEmitter;
+import com.aws.greengrass.telemetry.impl.Metric;
+import com.aws.greengrass.telemetry.impl.MetricFactory;
+
+import java.util.List;
+import javax.inject.Inject;
+
+public class MetricsEmitter extends PeriodicMetricsEmitter {
+    private static final String NAMESPACE = "ClientDeviceAuth";
+    private final MetricFactory mf = new MetricFactory(NAMESPACE);
+    private final ClientDeviceAuthMetrics metrics;
+
+    /**
+     * Constructor for metrics emitter.
+     *
+     * @param metrics {@link ClientDeviceAuthMetrics}
+     */
+    @Inject
+    public MetricsEmitter(ClientDeviceAuthMetrics metrics) {
+        super();
```

Comment:
nit: can remove, this isn't doing anything

Reply:
Isn't the constructor needed for injecting the metrics class now?

Reply:
yep, just talking specifically about `super()`

Reply:
the maven build requires me to call `super()` in the constructor so I think I'll have to leave it in for now

Reply:
ahh it's a pmd rule